libkbfs/md_journal.go

```diff
@@ -196,13 +196,15 @@ func makeMDJournalWithIDJournal(
 	}
 
 	if earliest != nil {
-		if earliest.BID() != latest.BID() {
+		if earliest.BID() != latest.BID() &&
+			!(earliest.BID() == NullBranchID &&
```

Comment:
I must be missing something, but when can this happen? I thought converting a branch to LocalSquashBranchID converted the whole branch...